plugins/org.jkiss.dbeaver.ext.ui.tipoftheday/tips.xml

```diff
@@ -1,95 +1,231 @@
-	<tip>In <a href="https://github.com/dbeaver/dbeaver/wiki/Data-Editor"> Data Editor</a>, press <b>Ctrl+F</b> to search for data in the current table.</tip>
-	<tip>You can generate SQL statements (SELECT/INSERT/UPDATE/DELETE) based on selected rows. To generate SQL, right-click the selected rows ,then click Generate SQL and select one of the SQL commands you see.</tip>
-	<tip>DBeaver allows viewing diagrams of existing tables and whole database schemas, see <a href="https://github.com/dbeaver/dbeaver/wiki/Database-Structure-Diagrams">ER diagrams</a> article.</tip>
-	<tip>DBeaver allows creating custom diagrams, see <a href="https://github.com/dbeaver/dbeaver/wiki/Custom-Diagrams">Custom Diagrams</a>.</tip>
-	<tip>DBeaver is a customizable software.<br/>For additional information on optional extensions, see <a href="https://github.com/dbeaver/dbeaver/wiki/Optional-extensions"> Optional Extensions</a></tip>
-	<tip>Use <a href="https://github.com/dbeaver/dbeaver/wiki/Mock-Data-Generation-in-DBeaver">DBeaver Mock Data Generator</a> when you need to create a huge quantities of realistic sample data for testing purposes and want to avoid tedious and time-consuming work of entering test data by hand.
-		<br/>Note: this tool is available in <a href="https://github.com/dbeaver/dbeaver/wiki/Enterprise-Edition">Enterprise</a> and <a href="https://github.com/dbeaver/dbeaver/wiki/Ultimate-Edition">Ultimate</a> editions only.
-	</tip>
-	<tip>You can customize the menu bar and the list of menu items displayed <a href="prefs://org.eclipse.ui.window.customizePerspective">here</a> in the <b>Menu Visibility</b> tab.</tip>
-	<tip>You can customize the toolbar <a href="prefs://org.eclipse.ui.window.customizePerspective">here</a> in the <b>Toolbar Visibility</b> tab.</tip>
-	<tip>To execute the selected SQL expression and print the results use the following shortcut: <b>Ctrl+Alt+'</b>.</tip>
+	<!--        Tips below were reviewed on 30th of June 2023   -->
+	<tip>DBeaver can be customized to suit your needs. Learn detailed information about optional extensions <a href="https://dbeaver.com/docs/wiki/Optional-extensions"> here</a>.</tip>
+	<tip>You can <a href="prefs://org.eclipse.ui.window.customizePerspective">customize the menu bar and the list of menu items displayed</a> in the <b>Menu Visibility</b> tab.</tip>
+	<tip>You can <a href="prefs://org.eclipse.ui.window.customizePerspective">customize the toolbar </a> in the <b>Toolbar Visibility</b> tab.</tip>
+	<tip>You can create a keyboard shortcut for most commands in DBeaver. <a href="prefs://org.eclipse.ui.preferencePages.Keys">Existing shortcuts can also be changed</a></tip>
+	<tip>You can change the current <a href="https://dbeaver.com/2022/05/26/change-the-data-in-the-smart-way/">database transaction mode</a> in the toolbar.
+		 Click the dropdown menu to the right of the <b>Commit and Rollback buttons</b>.</tip>
```

Comment:
Could you look if there are any cases when wiki has more recent snapshot?

Reply:
It's not a wiki. It an article in the blog. There is only one version of it.